build.cake

```diff
@@ -210,21 +211,38 @@ private void SignBinaries(string outputDirectory)
 {
     Information($"Signing binaries in {outputDirectory}");
 
-	var files = GetFiles(outputDirectory + "/Calamari.exe");
-    files.Add(GetFiles(outputDirectory + "/Calamari.dll"));
-
+    // check that any unsigned executables or libraries get signed, to play nice with security scanning tools
+    // refer: https://octopusdeploy.slack.com/archives/C0K9DNQG5/p1551655877004400
```

Comment:
consider adding "we are signing dll's we have written (& dont own), but we are asserting that they are distributed by us, and are not altered after production"